### PR TITLE
Updates related to the S40H modem launch

### PR DESCRIPTION
**Pull request by manuignatius**
Updated pages:
- https://subnero.com/website-staging/wnc/news/2024/03/01/Product-Launch-S40H-series-modems.html
- https://subnero.com/website-staging/products/
- https://subnero.com/website-staging/products/modem.html?&section=S40Hseries
- https://subnero.com/website-staging/products/wnc-s40hss4.html
- https://subnero.com/website-staging/products/wnc-s40hso4.html
- https://subnero.com/website-staging/products/wnc-s40hss4+xch.html

Existing issues:
- When clicked on "optional accessories" option on the top in modem page, the page doesn't display any images. This need to be fixed.
- Before final release, need to update the date on the pulse page to 11 March (currently using 1 March to render it properly)

- A few links in the pulse page and landing page will not work in staging website, but they will work in actual website as the links are pointed to the actual website.

### CODE REVIEW

**Review by AmyGibSubnero — DISMISSED** [review]

File: _posts/2024-03-01-Product-Launch-S40H-series-modems.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+*Singapore, March 11, 2024* – Subnero Pte. Ltd., a leading provider of underwater communication solutions headquartered in Singapore, proudly announces the launch of its latest innovation, the [S40H series](/products/modem.html?&section=S40Hseries) of high-speed acoustic smart modems, marking a significant advancement in wireless networked communication ([WNC](/products/wnc.html)) technology.
+
+<div class='pulse-img-div'>
+    <img src="{{site.baseurl}}/images/pulse-hf-pr.jpg" class='pulse-img'>
+</div>
+
+Designed to excel in challenging shallow water conditions, the S40H series introduces a groundbreaking blend of cutting-edge features:
+
+- **Enhanced Performance:** Utilizing a higher frequency band and boasting a larger bandwidth (25 kHz bandwidth, from 25 - 50 kHz band), the S40H series achieves data rates of up to 33 kbps, ensuring seamless connectivity even in the harshest underwater environments.
+- **Extended Range:** With the capability to provide connectivity for up to 1 km, these modems offer unparalleled communication range, enabling unprecedented access in challenging conditions.
```
Comment (esthertanhj) [inline _posts/2024-03-01-Product-Launch-S40H-series-modems.md]: My suggestion is to shorten the sentence and avoid the double use of the word "unparallelled" which is used for another point below.
```suggestion
- **Extended Range:** With the capability to provide wireless connectivity for a communication range up to 1 km, enabling unprecedented access in challenging conditions.
```

File: _posts/2024-03-01-Product-Launch-S40H-series-modems.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- **Versatile Applications:** Equipped with advanced underwater communication algorithms, the S40H series unlocks a myriad of possibilities, including high-speed data transfer for underwater IoT applications, underwater image transfer, and large sensor payload data transfer for marine robotics.
+- **Enhanced Performance:** The [multi-channel variant](/products/wnc-s40hss4+xch.html) of the modem further enhances performance through robust links supported by leveraging spatial diversity combining techniques.
+- **Powered by UnetStack:** Leveraging UnetStack, a versatile software framework for underwater communications and networking, the S40H series delivers unparalleled performance and adaptability.
```
Comment (esthertanhj) [inline _posts/2024-03-01-Product-Launch-S40H-series-modems.md]:
```suggestion
- **Powered by UnetStack:** Leveraging [UnetStack](https://unetstack.net/), a versatile software framework for underwater communications and networking, the S40H series delivers unparalleled performance and adaptability.
```

File: _posts/2024-03-01-Product-Launch-S40H-series-modems.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+*Singapore, March 11, 2024* – Subnero Pte. Ltd., a leading provider of underwater communication solutions headquartered in Singapore, proudly announces the launch of its latest innovation, the [S40H series](/products/modem.html?&section=S40Hseries) of high-speed acoustic smart modems, marking a significant advancement in wireless networked communication ([WNC](/products/wnc.html)) technology.
+
+<div class='pulse-img-div'>
+    <img src="{{site.baseurl}}/images/pulse-hf-pr.jpg" class='pulse-img'>
+</div>
+
+Designed to excel in challenging shallow water conditions, the S40H series introduces a groundbreaking blend of cutting-edge features:
+
+- **Enhanced Performance:** Utilizing a higher frequency band and boasting a larger bandwidth (25 kHz bandwidth, from 25 - 50 kHz band), the S40H series achieves data rates of up to 33 kbps, ensuring seamless connectivity even in the harshest underwater environments.
```
Comment (esthertanhj) [inline _posts/2024-03-01-Product-Launch-S40H-series-modems.md]: The term "Enhanced Performance" appears twice in the list of points below.

```suggestion
- **High Data Rates:** Utilizing a higher frequency band and boasting a larger bandwidth (25 kHz bandwidth, from 25 - 50 kHz band), the S40H series achieves data rates of up to 33 kbps, ensuring seamless connectivity even in the harshest underwater environments.
```

File: _posts/2024-03-01-Product-Launch-S40H-series-modems.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- **Extended Range:** With the capability to provide connectivity for up to 1 km, these modems offer unparalleled communication range, enabling unprecedented access in challenging conditions.
+- **Versatile Applications:** Equipped with advanced underwater communication algorithms, the S40H series unlocks a myriad of possibilities, including high-speed data transfer for underwater IoT applications, underwater image transfer, and large sensor payload data transfer for marine robotics.
+- **Enhanced Performance:** The [multi-channel variant](/products/wnc-s40hss4+xch.html) of the modem further enhances performance through robust links supported by leveraging spatial diversity combining techniques.
```
Comment (esthertanhj) [inline _posts/2024-03-01-Product-Launch-S40H-series-modems.md]: Proposed another word as the term "performance" appears twice in this list of points.
```suggestion
- **Enhanced Capability:** The [multi-channel variant](/products/wnc-s40hss4+xch.html) of the modem further enhances performance through robust links supported by leveraging spatial diversity combining techniques.
```

File: _posts/2024-03-01-Product-Launch-S40H-series-modems.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- **Powered by UnetStack:** Leveraging UnetStack, a versatile software framework for underwater communications and networking, the S40H series delivers unparalleled performance and adaptability.
+- **Class-Leading Performance:** Combining a software-defined open architecture design with edge computing capability and machine learning-based link tuning features, these modems set a new standard for performance in the subsea wireless industry.
+
+The S40H series modems are officially launched during Oceanology International 2024, with immediate availability for orders. Shipping will commence during the summer of 2024.
+
+*About Subnero:* Subnero serves defense, marine, subsea, offshore energy, and scientific research sectors with global customers, enabling a range of use cases in the most demanding conditions. For more information, visit [Subnero's website](https://subnero.com).
+
+For media inquiries, please contact sales@subnero.com
```
Comment (esthertanhj) [inline _posts/2024-03-01-Product-Launch-S40H-series-modems.md]:
```suggestion
For media inquiries, please contact sales@subnero.com.
```

**Comment by esthertanhj** [comment]
I guess the main [WNC site](https://subnero.com/website-staging/products/wnc.html) needs to be updated to include the S40H modems brochure, and a sentence to state that our acoustic smart modems product line contains both S40H and M25M series.

File: products/wnc-s40hss4+xch.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+- Large sensor payload data transfer for marine robotics
+- Underwater networks
+- Rapid environmental assessment.
+- Underwater localization & positioning (e.g. USBL)
+
+## Key Features
+
+- Multiple synchronized receiving channels for enhanced communication performance.
+- 2x communication performance as compared to the [single channel]({{site.baseurl}}/products/wnc-s40hss4.html) equivalents.
+- Ability to choose between 1 - 4 additional receiving channels.
```
Comment (esthertanhj) [inline products/wnc-s40hss4+xch.md]: Would this be 3 additional receiving channels for standard production (as Project Selene is a customisation)?
```suggestion
- Ability to choose between 1 - 3 additional receiving channels.
```

**Comment by manuignatius** [comment]
> WNC site

Added the brochure to WNC page. As for the mention, it is already mentioned in the "best in class" section. Will add a prominent section once we have baleine too out.

**Review by esthertanhj — DISMISSED** [review]

**Comment by AmyGibSubnero** [comment]
Comments:
- I find this sentence on the XCH page waaaay too long as an intro - `Engineered with cutting-edge spatial diversity combining techniques to optimize data extraction across multiple receiving channels, Subnero’s multi-channel configuration smart modems redefine underwater communication performance, delivering unparalleled reliability in the most demanding environments.` Can we shorten it a bit?
- The graph "Enhanced communication performance" Y-Axis title doesn't make sense. Its comms success rate right? Else the SC would have a 77% performance improvement vs what?
- The intro paragraphs on SC and OEM are the same, do we want to differentiate and market OEM as designed for integration? Or to explain what OEM is vs the series of modem?
- on the product launch page, can we bold or change the colour of the leading words in the bullet points e.g. "Enhanced Performance"
- This sentence sounds a bit idk `With the capability to provide connectivity for up to 1 km, these modems offer unparalleled communication range, enabling unprecedented access in challenging conditions.` I think it is the 'unparalleled' followed by the 'unprecedented'. Would go with something more like `Providing connectivity of up to 1 km, this series offers a competitive range in challenging conditions to enable your short-range deployments.'
- We repeat the title of "Enhanced Performance" in bullet point 4. 
- Replace `The S40H series modems are officially launched during Oceanology International 2024` with `Subnero is officially launching the S40H series modems at Oceanology International 2024` and may end with our booth number.

**Review by AmyGibSubnero — COMMENTED** [review]
Comments:

- I find this sentence on the XCH page waaaay too long as an intro - `Engineered with cutting-edge spatial diversity combining techniques to optimize data extraction across multiple receiving channels, Subnero’s multi-channel configuration smart modems redefine underwater communication performance, delivering unparalleled reliability in the most demanding environments.` Can we shorten it a bit?
- The graph "Enhanced communication performance" Y-Axis title doesn't make sense. Its comms success rate right? Else the SC would have a 77% performance improvement vs what?
- The intro paragraphs on SC and OEM are the same, do we want to differentiate and market OEM as designed for integration? Or to explain what OEM is vs the series of modem?
- On the product launch page, can we bold or change the colour of the leading words in the bullet points e.g. "Enhanced Performance"
- This sentence sounds a bit odd `With the capability to provide connectivity for up to 1 km, these modems offer unparalleled communication range, enabling unprecedented access in challenging conditions.` I think it is the 'unparalleled' followed by the 'unprecedented'. Would go with something more like `Providing connectivity of up to 1 km, this series offers a competitive range in challenging conditions to enable your short-range deployments.`
- We repeat the title of "Enhanced Performance" in bullet point 4.
- Replace The S40H series modems are officially launched during Oceanology International 2024 with Subnero is officially launching the S40H series modems at Oceanology International 2024 and may end with our booth number.

**Comment by manuignatius** [comment]
> Comments:
> 
> * I find this sentence on the XCH page waaaay too long as an intro - `Engineered with cutting-edge spatial diversity combining techniques to optimize data extraction across multiple receiving channels, Subnero’s multi-channel configuration smart modems redefine underwater communication performance, delivering unparalleled reliability in the most demanding environments.` Can we shorten it a bit?

- Proposals welcome, I tried a few, but find this still the best.

> * The graph "Enhanced communication performance" Y-Axis title doesn't make sense. Its comms success rate right? Else the SC would have a 77% performance improvement vs what?

- Updated to read "Communication performance" for y-axis

> * The intro paragraphs on SC and OEM are the same, do we want to differentiate and market OEM as designed for integration? Or to explain what OEM is vs the series of modem?

- Yes I agree this is a bit weird, but the intro para is about silver edition. The differentiation between SC and OC comes later in the page. Unfortunately, this is to do with our current website layout and it is the same for M25M modems too. I am not modifying it for the time being (to avoid going down the rabbit hole). We can address all of these during website phase 2

> * On the product launch page, can we bold or change the colour of the leading words in the bullet points e.g. "Enhanced Performance"

- It is bold, but it is noticeable due to our theme. Will go with the theme for now.

> * This sentence sounds a bit odd `With the capability to provide connectivity for up to 1 km, these modems offer unparalleled communication range, enabling unprecedented access in challenging conditions.` I think it is the 'unparalleled' followed by the 'unprecedented'. Would go with something more like `Providing connectivity of up to 1 km, this series offers a competitive range in challenging conditions to enable your short-range deployments.`

- Updated

> * We repeat the title of "Enhanced Performance" in bullet point 4.

- Updated to remove the repeat

> * Replace The S40H series modems are officially launched during Oceanology International 2024 with Subnero is officially launching the S40H series modems at Oceanology International 2024 and may end with our booth number.

- Updated

**Review by esthertanhj — APPROVED** [review]

**Review by AmyGibSubnero — APPROVED** [review]

**Pull request merged**